build_tools/docker/py311.dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM python:3.11.2-bullseye
+FROM python:3.11.8-slim-bookworm
```

Comment:
It's not a good idea to mention the patch version, as mentioned by @yarnabrina. Simply using `python:3.11-slim-bookworm` will suffice.